packages/ui/src/components/Alert/Alert.stories.tsx

```diff
@@ -25,32 +25,50 @@ const TemplateAlert: ComponentStory<typeof Alert.Root> = ({
   children,
   ...args
 }) => {
+  const Alerts = types.map((type) => (
+    <div key={type}>
+      <React.Fragment key={type}>
+        <Alert.Root {...args} type={type}>
+          <Alert.Title>
+            {type.charAt(0).toUpperCase() + type.slice(1)}
+          </Alert.Title>
+          <p>{children}</p>
+          {type === "error" && (
+            <Alert.Footer>
+              <p data-h2-font-size="base(caption)">
+                <Link external href="/#">
+                  Reach out to our support team
+                </Link>{" "}
+                if you have any questions.
+              </p>
+            </Alert.Footer>
+          )}
+        </Alert.Root>
+      </React.Fragment>
+    </div>
+  ));
   return (
-    <div
-      data-h2-padding="base(x2)"
-      data-h2-display="base(flex)"
-      data-h2-flex-direction="base(column)"
-    >
-      {types.map((type) => (
-        <React.Fragment key={type}>
-          <Alert.Root {...args} type={type}>
-            <Alert.Title>
-              {type.charAt(0).toUpperCase() + type.slice(1)}
-            </Alert.Title>
-            <p>{children}</p>
-            {type === "error" && (
-              <Alert.Footer>
-                <p>
-                  <Link external href="/#">
-                    Reach out to our support team
-                  </Link>{" "}
-                  if you have any questions.
-                </p>
-              </Alert.Footer>
-            )}
-          </Alert.Root>
-        </React.Fragment>
-      ))}
+    <div>
+      <div
+        data-h2-display="base(grid) base:children[>div>div](grid)"
+        data-h2-grid-template-columns="base(1fr) base:children[>div>div](repeat(2, minmax(0, 1fr)))"
+        data-h2-padding="base:children[>div>div](x2)"
+        data-h2-gap="base:children[>div>div](x1)"
+        data-h2-background-color="base:children[>div>div](background)"
```

Comment:
I really like the new format of story showing all the different variants together!  I don't love all this boilerplate in teh story but hopefully we can eventually figure out how to factor it out to something reusable.

Reply:
I agree, boilerplate is kinda gross, but at the same time, it's not 100% the same across stories. Some components require more vertical/horizontal space, so the grid changes based on their individual needs. Open to ways of trying to standardize this though!